src/core/utils.js

```diff
@@ -12,12 +12,27 @@ const Stream = require('stream');
 module.exports = {
 
     lang(filePath) {
-        return fang(filePath) || {
-            name: Path.parse(filePath).ext.replace('.', '').toUpperCase(),
-            mode: 'text',
-            scope: null,
-            color: null,
-        };
+        const name = Path.parse(filePath).ext.replace('.', '').toUpperCase();
+        switch (name) {
+            case 'NUNJUCKS':
+            case 'NUNJS':
+            case 'NUNJ':
+            case 'NJ':
+            case 'J2':
+                return {
+                    name: 'HTML+Django',
+                    mode: 'django',
+                    scope: 'text.html.django',
+                    color: null,
+                };
+            default:
+                return fang(filePath) || {
+                    name: name,
+                    mode: 'text',
```

Comment:
According to the issue we're having [here](https://github.com/frctl/fractal/issues/576), Highlight.js now requires this to be `plaintext`.

Reply:
Good point, I'll have a look at this tomorrow.